src/main/java/selenium/pages/LoginPageWithPF.java

```diff
@@ -0,0 +1,28 @@
+package selenium.pages;
+
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import selenium.Properties;
+import selenium.TestUser;
+
+public class LoginPageWithPF extends BasePage {
+    @FindBy(id = "user-name")
+    private WebElement usernameInput;
+
+    @FindBy(id = "password")
+    private WebElement passwordInput;
+
+    @FindBy (id = "login-button")
+    private WebElement loginButton;
+
+    public void openPage() {
+        this.driver.get(Properties.LOGIN_URL);
+    }
```

Comment:
What does this method do? Does it open the app? First of all this method should return some page, LoginPage as far as I understood, also this method better to move to a base test class

Reply:
Updated